lib/blocs/authentication/viewmodels/authentication_event.dart

```diff
@@ -4,7 +4,7 @@ abstract class AuthenticationEvent extends Equatable {
   const AuthenticationEvent();
 
   @override
-  List<Object> get props => [];
+  List<Object?> get props => [];
```

Comment:
why are these needed ? Something changed in the overriden class?

Reply:
Not really, what happens is that we have variables like the `pageCommand` that are nullable and that is why it has to be set to accept objects of nullable type, since we do this frequently I thought it would be good to change all of them to accepting nullables at once, really It shouldn't affect anything.